airrohr-firmware/PCF8574.h

```diff
@@ -120,6 +120,7 @@ class PCF8574
     PCF8574(TwoWire *pWire, uint8_t address, uint8_t sda, uint8_t scl, uint8_t interruptPin, void (*interruptFunction)());
 #endif
 
+    void setAddress(uint8_t address);
```

Comment:
@Warukira we should not be adding methods not included in the original library

Reply:
@chegejames ,okay. Fixing this.